vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -38,3 +42,89 @@ App::App() noexcept
 
     InitializeComponent();
 }
+
+/// <summary>
+/// Invoked when the application is launched normally by the end user.  Other entry points
+/// will be used such as when the application is launched to open a specific file.
+/// </summary>
+/// <param name="e">Details about the launch request and process.</param>
+void App::OnLaunched(LaunchActivatedEventArgs const& e)
```

Comment:
>OnLaunched [](start = 10, length = 10)

I wonder why the C++ OnLaunch code is so much more complex than C#.

Reply:
not sure, but we could bring it up with the cppwinrt alias if you'd like to find out :)